hubspot/plugin.py

```diff
@@ -1701,19 +1022,15 @@ def hubspot_search_deals(
     dict. You MUST check whether the property exists in additional_properties before using it.
 
     Args:
-        return_with_custom_properties: A sequence of custom property names to fetch from found
-            deals. These will be included in additional_properties if they exist.
-
+        return_with_custom_properties: Deprecated. 
```

Comment:
I think we should just remove `return_with_custom_propertiess`. 

We don't have alot of users actively using HubSpot it would be easier for code_gen moving forward. 

I can reach out to the user to let them know they need to regenerate the code if they helps.

@jngiam might have more to say here

Reply:
yeah, this seems like a good idea if we're okay with any workflow breakage!

Reply:
yeah i think it's ok break some things here for now